src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py

```diff
@@ -38,24 +48,68 @@ def __init__(self, id: str, post_object: GraphicsDefn):
             if in_notebook() or get_config()["blocking"]
             else BackgroundPlotter(title=f"PyFluent ({self.id})")
         )
+        self.overlay: bool = False
+        self.fetch_data: bool = False
+        self.show_window: bool = True
         self.animate: bool = False
         self.close: bool = False
         self.refresh: bool = False
         self.update: bool = False
         self._visible: bool = False
         self._init_properties()
+        self._data = {}
+        self._colors = {
```

Comment:
@ajain-work do these keys have a practical purpose?

Reply:
No. Currently we are coloring automatically. But these will be needed if we do manual coloring.